raspiraw.c

```diff
@@ -946,6 +1186,22 @@ int main(int argc, const char** argv) {
 						brcm_header->mode.bayer_format = VC_IMAGE_BAYER_RAW16;
 						break;
 				}
+				if (cfg.write_header0)
```

Comment:
Take in a new filename again rather than squeezing it in as file 0. The user can then choose to name it as file 0, or use something more to their liking.